Kanso UI tour (est. pub Feb 15) (#3424)
SumoLogic/sumologic-documentation: docs/get-started/sumo-logic-ui.md

### Get Help: Docs, community, and more

There are a number of places where you can get the answers to questions you have:

* Check out our [Release Notes](/docs/release-notes)
* Search [Sumo Docs](https://help.sumologic.com/)
* Visit the **Learn Page** in the Sumo Logic UI
* Post a question on the [Sumo Logic Community](https://support.sumologic.com/hc/en-us/community/topics)
* Contact [Support](https://support.sumologic.com/)
* Join our [Sumo Logic Slack](https://sumodojo.slack.com/) channel

:::tip
See [Help Resources and Contacts](/docs/get-started/help) for full information.
:::

## Administrator tasks

### Manage data collection, data settings, and alerts

Sumo Logic Administrators (Admins) are responsible for managing data collection, data settings, and alerts for their organization. You must have Sumo Logic Admin role privileges to perform these tasks.

To manage data in Sumo Logic, do the following:

1. From the top-right navigation bar, click **Configuration**.<br/><img src={useBaseUrl('img/get-started/config.png')} alt="img/get-started/config.png" width="400"/>
1. Choose from the following, as needed:

>     * **Collection**. Manage [Collectors and Sources](/docs/send-data/collection), [OpenTelemetry Collection](/docs/send-data/opentelemetry-collector), [Data Collection Status](/docs/manage/ingestion-volume/collection-status-page), [Ingest Budget](/docs/manage/ingestion-volume/ingest-budgets), Archive, and [Data Archiving](/docs/manage/data-archiving).

Reviewer: ```suggestion
    * **Collection**. Manage [collection](/docs/send-data/collection), [OpenTelemetry Collection](/docs/send-data/opentelemetry-collector), [Data Collection Status](/docs/manage/ingestion-volume/collection-status-page), [Ingest Budget](/docs/manage/ingestion-volume/ingest-budgets), Archive, and [Data Archiving](/docs/manage/data-archiving).
```

>     * **Logs**. Manage [fields](/docs/manage/fields), [field extraction rules](/docs/manage/field-extractions), [parsers](/docs/search/search-query-language/parse-operators), [partitions](/docs/manage/partitions-data-tiers), [scheduled views](/docs/manage/scheduled-views), and [data forwarding](/docs/manage/data-forwarding).

Reviewer: ```suggestion
    * **Logs**. Manage [fields](/docs/manage/fields), [field extraction rules](/docs/manage/field-extractions), [parsers](/docs/cse/schema/parser-editor/), [partitions](/docs/manage/partitions-data-tiers), [scheduled views](/docs/manage/scheduled-views), and [data forwarding](/docs/manage/data-forwarding).
```

Author: @jpipkin1 , I think since this is in the Logs section, it's referring to log search parsers.

    * **Metrics**. Manage [metrics rules](/docs/metrics/metric-rules-editor), [logs-to-metrics](../metrics/logs-to-metrics.md), and [metrics transformation rules](../metrics/metrics-transformation-rules.md).
    * **Cloud SIEM Integrations**. Manage [Ingest Mappings](/docs/cse/ingestion/sumo-logic-ingest-mapping), [Log Mappings](/docs/cse/schema/create-structured-log-mapping), [Context Actions](/docs/cse/administration/create-cse-context-actions), [Actions](/docs/cse/administration/create-cse-actions), [Enrichment](/docs/cse/integrations/enrichments-and-indicators), [Automation](/docs/cse/automation).

>     * **Cloud SIEM Entities**. Manage [Groups](/docs/cse/records-signals-entities-insights/create-an-entity-group), [Normalization](/docs/cse/schema/username-and-hostname-normalization), [Custom Types](/docs/cse/records-signals-entities-insights/create-custom-entity-type), and [Criticality](/docs/cse/records-signals-entities-insights/entity-criticality).

Reviewer: ```suggestion
    * **Cloud SIEM Entities**. Manage [Groups](/docs/cse/records-signals-entities-insights/create-an-entity-group), [Normalization](/docs/cse/records-signals-entities-insights/configure-entity-lookup-table/), [Custom Types](/docs/cse/records-signals-entities-insights/create-custom-entity-type), and [Criticality](/docs/cse/records-signals-entities-insights/entity-criticality).
```

>     * **Cloud SIEM Workflow**. Manage [Insight Detection](/docs/cse/records-signals-entities-insights/set-insight-generation-window-threshold), [Insight Statuses](/docs/cse/administration/manage-custom-insight-statuses), [Insight Resolutions](/docs/cse/administration/manage-custom-insight-resolutions), and [Tag Schemas](/docs/cse/administration/create-a-custom-tag-schema).

Author: Hi @jpipkin1 , when you have the bandwidth, would you be able to review the above docs links to make sure they accurately reflect those topics? I did it from scratch and not 100% sure if it's right.

Reviewer: Beautiful work, @kimsauce! I only found a few things to update.

>     * **Cloud SIEM Workflow**. Manage [Insight Detection](/docs/cse/records-signals-entities-insights/set-insight-generation-window-threshold), [Insight Statuses](/docs/cse/administration/manage-custom-insight-statuses), [Insight Resolutions](/docs/cse/administration/manage-custom-insight-resolutions), and [Tag Schemas](/docs/cse/administration/create-a-custom-tag-schema).

Reviewer: ```suggestion
    * **Cloud SIEM Workflow**. Manage [Insight Detection](/docs/cse/records-signals-entities-insights/set-insight-generation-window-threshold), [Insight Statuses](/docs/cse/administration/manage-custom-insight-statuses), [Insight Resolutions](/docs/cse/administration/manage-custom-insight-resolutions), and [Tag Schemas](/docs/cse/administration/create-a-custom-tag-schema).
    * **Cloud SOAR Configurations**. Manage [Incidents](/docs/cloud-soar/incidents-triage/), [Fields](/docs/cloud-soar/incidents-triage/#custom-fields), and [Incident Labels](/docs/cloud-soar/global-functions-menu/#incident-label).
```

### Manage accounts, users, and security

Sumo Logic administrators (admins) manage user accounts, user roles, and security. You must have Sumo Logic Admin role privileges to perform these tasks.

To administer Sumo Logic accounts, users, and security, do the following:

1. From the top navigation bar, click **Administration**.<br/><img src={useBaseUrl('img/get-started/admin.png')} alt="img/get-started/admin.png" width="400"/>
1. Choose from any of the following:
    * **Account**. [View information about your organization's Sumo Logic subscription](/docs/manage/manage-subscription), [enable and manage the data volume index,](/docs/manage/ingestion-volume/data-volume-index) [manage billing](/docs/manage/manage-subscription).
    * **Users and Roles**. [Manage users and roles](/docs/manage/users-roles).

>     * **Account Security Settings**. [Set password policy for your org](/docs/manage/security/set-password-policy), [set up security whitelist](/docs/manage/security/create-allowlist-ip-cidr-addresses), [manage access keys](/docs/manage/security/access-keys), manage security policies ([audit index](/docs/manage/security/audit-indexes/audit-index)), [support account access](/docs/manage/security/enable-support-account), [dashboard sharing](/docs/dashboards/share-dashboard-new/), and [set up SAML authentication](/docs/manage/security/saml).<br/><img src={useBaseUrl('img/get-started/admin.png')} alt="img/get-started/admin-expanded.png" width="400"/>

Reviewer: ```suggestion
    * **Account Security Settings**. [Set password policy for your org](/docs/manage/security/set-password-policy), [set up security whitelist](/docs/manage/security/create-allowlist-ip-cidr-addresses), [manage access keys](/docs/manage/security/access-keys), manage security policies ([audit index](/docs/manage/security/audit-indexes/audit-index)), [support account access](/docs/manage/security/enable-support-account), [dashboard sharing](/docs/dashboards/share-dashboard-new/), and [set up SAML authentication](/docs/manage/security/saml).<br/><img src={useBaseUrl('img/get-started/admin.png')} alt="img/get-started/admin-expanded.png" width="400"/>
    * **Cloud SOAR Settings**. Manage [general settings](/docs/cloud-soar/global-functions-menu/#general-settings), [Notifications](/docs/cloud-soar/global-functions-menu/#notifications), and [Groups](/docs/cloud-soar/global-functions-menu/#groups).
```

Reviewer: @kimsauce, I will go ahead and commit this change to add "Cloud SOAR Settings" so we don't forget about it.

Author: thank you!

* Recently Opened Dashboards
* Recently Run Searches
* Recommended Dashboards 
* Pinned Searches

<img src={useBaseUrl('img/get-started/dashboard-searches.png')} alt="dashboard-searches.png" />

### Access dashboards and searches

The left nav panel provides easy access to libraries, searches, folders, and your personal collection of dashboards. Click the icons at the top of the left-side navigation panel to view:

* Your **Library**, which contains:
   * Your **Personal** dashboards and searches.
   * Your own **Installed Apps** from the App Catalog.
   * Dashboards and searches shared within your organization.<br/><img src={useBaseUrl('/img/get-started/library.png')} alt="library.png" width="500"/>
* **Recent** dashboards and searches.<br/><img src={useBaseUrl('/img/get-started/recent.png')} alt="recent.png" width="350"/>
* Your **Favorites** list of favorited dashboards and searches.<br/><img src={useBaseUrl('/img/get-started/favs.png')} alt="favs.png" width="350"/>

<!--
### Search and switch browsing modes

The top of the left-side navigation panel is where you can search for content and
users and easily switch browsing modes. 

> * Searching. Enter text in the **Search** field to quickly find apps, dashboards, searches, and users. 
> * Switching browsing modes. Click the **Details** icon and make a selection from the dropdown menu.

Author: ```suggestion
* Searching—Enter text in the **Search** field to quickly find apps, dashboards, searches, and users. 
* Switching browsing modes—Click the **Details** icon and make a selection from the dropdown menu.
```

> ![TUI_Search_Details.png](/img/get-started/TUI_Search_Details.png)

Author: @agaur It looks like [this functionality](https://help.sumologic.com/docs/get-started/sumo-logic-ui/#searchand-switchbrowsing-modes) doesn't exist in the new UI. Can you confirm?

> -->

Author: @agaur It looks like [this functionality](https://help.sumologic.com/docs/get-started/sumo-logic-ui/#searchand-switchbrowsing-modes) doesn't exist in the new UI. Can you confirm?

> -->

Author: @agaur It looks like [this functionality](https://help.sumologic.com/docs/get-started/sumo-logic-ui/#searchand-switchbrowsing-modes) doesn't exist in the new UI. Can you confirm?

Reviewer 2: That's correct. It's not available in side navigation. You should go to library to search for saved content. 
(For more context, we are working on building spotlight search functionality which is going to be replacement of this)

    * **Metrics**. Manage metrics rules, [logs-to-metrics](../metrics/logs-to-metrics.md), and [metrics transformation rules](../metrics/metrics-transformation-rules.md).
    * **Cloud SIEM Integrations**.
    * **Cloud SIEM Entities**.
    * **Cloud SIEM Workflow**.

### Manage accounts, users, and security

Sumo Logic administrators (admins) manage user accounts, user roles, and security. You must have Sumo Logic Admin role privileges to perform these tasks.

To administer Sumo Logic accounts, users, and security, do the following:

1. From the top navigation bar and click **Administration**.<br/> <img src={useBaseUrl('img/get-started/admin.png')} alt="img/get-started/admin.png" width="400"/>
1. Choose from the following, as needed: <!-- what does this look like for Admin users? -->
    * **Account**. [View information about your organization's Sumo Logic subscription](/docs/manage/manage-subscription), [enable and manage the data volume index,](/docs/manage/ingestion-volume/data-volume-index) [manage billing](/docs/manage/manage-subscription).
    * **Users and Roles**. [Manage users and roles](/docs/manage/users-roles).

>     * **Security**. [Set password policy for your org](/docs/manage/security/set-password-policy), [set up security whitelist](/docs/manage/security/create-allowlist-ip-cidr-addresses), [manage access keys](/docs/manage/security/access-keys), manage security policies ([audit index](/docs/manage/security/audit-indexes/audit-index)), [support account access](/docs/manage/security/enable-support-account), [dashboard sharing](../dashboards/share-dashboard-new), and [set up SAML authentication](/docs/manage/security/saml).

Author: @agaur Can you share what options appear beneath **Administration**? I don't have Admin access, so I'm seeing limited options:
<img width="300" alt="Screenshot 2023-12-26 at 4 25 17 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/cfced5c5-75a2-4782-b4f3-38e37f0c0684">

Reviewer: @kimsauce, here is what I see:
<img width="273" alt="Screenshot 2024-01-03 at 3 13 16 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/127961815/7c1a1c09-b602-487c-b8c2-c81079138d0d">

Author: @jpipkin1 , oh wow, that's totally different. It looks like you're on stag and I'm on cust. usage. @agaur can you clarify?

Reviewer 2: Hi @kimsauce, 
Yes, the items that you see in the admin menu will be based on your permissions. This behaviour has not changed from Bento. Whatever administration pages the user had access to in Bento, they will be accessible in Kanso as well. That's why the different list on stagData and customerUsage because user permissions are different.

>     * **Security**. [Set password policy for your org](/docs/manage/security/set-password-policy), [set up security whitelist](/docs/manage/security/create-allowlist-ip-cidr-addresses), [manage access keys](/docs/manage/security/access-keys), manage security policies ([audit index](/docs/manage/security/audit-indexes/audit-index)), [support account access](/docs/manage/security/enable-support-account), [dashboard sharing](../dashboards/share-dashboard-new), and [set up SAML authentication](/docs/manage/security/saml).

Reviewer: ```suggestion
    * **Security**. [Set password policy for your org](/docs/manage/security/set-password-policy), [set up security whitelist](/docs/manage/security/create-allowlist-ip-cidr-addresses), [manage access keys](/docs/manage/security/access-keys), manage security policies ([audit index](/docs/manage/security/audit-indexes/audit-index)), [support account access](/docs/manage/security/enable-support-account), [dashboard sharing](/docs/get-started/dashboards/share-dashboard-new/), and [set up SAML authentication](/docs/manage/security/saml).
```

>     * **Security**. [Set password policy for your org](/docs/manage/security/set-password-policy), [set up security whitelist](/docs/manage/security/create-allowlist-ip-cidr-addresses), [manage access keys](/docs/manage/security/access-keys), manage security policies ([audit index](/docs/manage/security/audit-indexes/audit-index)), [support account access](/docs/manage/security/enable-support-account), [dashboard sharing](/docs/get-started/dashboards/share-dashboard-new/), and [set up SAML authentication](/docs/manage/security/saml).

Reviewer: ```suggestion
    * **Security**. [Set password policy for your org](/docs/manage/security/set-password-policy), [set up security whitelist](/docs/manage/security/create-allowlist-ip-cidr-addresses), [manage access keys](/docs/manage/security/access-keys), manage security policies ([audit index](/docs/manage/security/audit-indexes/audit-index)), [support account access](/docs/manage/security/enable-support-account), [dashboard sharing](/docs/dashboards/share-dashboard-new/), and [set up SAML authentication](/docs/manage/security/saml).
```
